integration-tests/src/test/kotlin/com/google/devtools/ksp/test/OnErrorIT.kt

```diff
@@ -105,8 +105,6 @@ class OnErrorIT {
                     it.startsWith("e: [ksp] kotlin.io.FileAlreadyExistsException:")
                 }
             )
-
-            Assert.assertTrue(result.output.contains("e: java.lang.IllegalStateException: Should not be called!"))
```

Comment:
Was this from AA, and somehow the path to that is fixed?

Reply:
This is testing for old implementation, which was fixed later in upstream.